README.md

```diff
@@ -175,7 +175,7 @@ Here are some benefits of using River (and online machine learning in general):
 
 ## 👍 Contributing
 
-Feel free to contribute in any way you like, we're always open to new ideas and approaches. You can also take a look at the [issue tracker](https://github.com/online-ml/river/issues) and the [icebox](https://github.com/online-ml/river/projects/2) to see if anything takes your fancy. Please check out the [contribution guidelines](https://github.com/online-ml/river/blob/master/CONTRIBUTING.md) if you want to bring modifications to the code base. You can view the list of people who have contributed [here](https://github.com/online-ml/river/graphs/contributors).
+Feel free to contribute in any way you like, we're always open to new ideas and approaches. You can also take a look at the [issue tracker](https://github.com/online-ml/river/issues) and the [discussions](https://github.com/online-ml/river/discussions) to see if anything takes your fancy. Please check out the [contribution guidelines](https://github.com/online-ml/river/blob/master/CONTRIBUTING.md) if you want to bring modifications to the code base. You can view the list of people who have contributed [here](https://github.com/online-ml/river/graphs/contributors).
```

Comment:
I think you should put a link to the board and not the discussions, no? The idea for the board is that contributers can take a look at what needs doing and hasn't been assigned yet. The discussions are there to discuss new ideas. What do you think?

Reply:
Sounds good to me. I wanted to point to the discussions somewhere since we already have that badge for the roadmap.

But that works too. Let me change it.

Reply:
You're right it's already linked somewhere, but we may as well overcommunicate a little so people don't miss it :)